docs/analytics.md

```diff
@@ -83,6 +83,7 @@ input_parameters = workflow.construct_parameters(
 )
 input_parameters["esa-s2-l2a-gtiff-visual:1"].update({"max_cloud_cover": 5})
 ```
+If parameters are not adjusted with `workflow.construct_parameters()`, they will be added as the default block parameter values.
```

Comment:
No, it's not true — e.g., if you don't submit an asset ID for a processing from storage block, nothing will work. There are no fallback IDs for it.